$OpenBSD: patch-configure,v 1.2 2011/03/07 19:27:27 ajacoutot Exp $

From e118a01dfefa589ce83fe43b13c5eb2732a0fcfb Mon Sep 17 00:00:00 2001
From: Matthew Barnes <mbarnes@redhat.com>
Date: Wed, 11 Aug 2010 14:26:06 +0000
Subject: Remove -DGDK_DISABLE_DEPRECATED
Until we can port the GDK drawing code to cairo.

--- configure.orig	Sun Feb  6 05:12:31 2011
+++ configure	Mon Feb  7 08:06:17 2011
@@ -5763,7 +5763,6 @@ if test "x${enable_deprecated_flags}" = "xyes"; then
 
   list="-DG_DISABLE_DEPRECATED
 		-DPANGO_DISABLE_DEPRECATED
-		-DGDK_DISABLE_DEPRECATED
 		-DGDK_PIXBUF_DISABLE_DEPRECATED
 		-DGTK_DISABLE_DEPRECATED
 		-DGNOME_DISABLE_DEPRECATED"
